python/pyngraph/ops/batch_norm.cpp

```diff
@@ -41,7 +41,8 @@ void regclass_pyngraph_op_BatchNormBackprop(py::module m)
                std::shared_ptr<ngraph::op::BatchNormBackprop>,
                ngraph::op::util::RequiresTensorViewArgs>
         batch_norm_backprop(m, "BatchNormBackprop");
-    batch_norm_backprop.doc() = "ngraph.impl.op.BatchNormBackprop wraps ngraph::op::BatchNormBackprop";
+    batch_norm_backprop.doc() =
```

Comment:
Does our `clang-format` checks the `python` directory as well?

Reply:
yup! It does [LINK](https://github.com/NervanaSystems/ngraph/blob/master/maint/apply-code-format.sh#L47)